### ISSUE TITLE
Cannot reenable video-track after disabling it

### ISSUE THREAD

**Issue by gpavlov21** [blackuy/react-native-twilio-video-webrtc#579]
### Steps to reproduce
1. Follow the example app 1:1
2. Add a function toggleVideo():
```
const toggleVideo = () => {
    if(twilioVideo && twilioVideo.current){
       twilioVideo.current.setLocalVideoEnabled(!isVideoEnabled).then(isEnabled => {
          setIsVideoEnabled(isEnabled)
          })
       }
    }
```

### Expected behaviour
The video should stop/start

### Actual behaviour
When I toggle off  the video it stops for me and for the remote participant as well, but when I toggle it back ON, it goes on only for me, for the remote participant(web browser), the video-track is not being enabled. 
### Environment
- **React Native version**: 0.63.4

**Comment by mumer-saleem** [comment]
same issue am also facing @gpavlov21

**Comment by gpavlov21** [comment]
@mumer-saleem On Android it still doesn't work

**Comment by mumer-saleem** [comment]
@gpavlov21  did you find solution

**Comment by gpavlov21** [comment]
@mumer-saleem no im still struggling to find a solution, almost 1 month later ....

**Comment by hpardess** [comment]
I am facing same issue... any solution??

**Comment by gpavlov21** [comment]
No, still no solution. After reviewing all the issues on github, i think this problem is around 2 years old .... @slycoder any idea ?

**Comment by slycoder** [comment]
Hi all, sorry I haven't been able to actively work on it.  If someone has a trace of events, that might help debug the issue.  For example, what enable/disable publish/unpublish events is the remote participant seeing?

**Comment by tzapevalov-tofi** [comment]
I'm getting this issue too and for me, remote participant is seeing 'disabled' event on the RemoteVideoTrack, and then 'started' with a new `sid`. Comparing this to when two browsers are performing the same actions which work, in that case the remote participant sees 'disabled' on the RemoteVideoTrack and then 'enabled' with the same `sid`.

**Comment by tzapevalov-tofi** [comment]
I managed to fix it by removing this line https://github.com/blackuy/react-native-twilio-video-webrtc/blob/0231f730b049464c2b447acdce63341e61c6dcc9/android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java#L720
I don't know what issues this may cause, but now my remote participant is seeing the expected events 'disabled' followed by 'enabled'.
